pkg/policy/repository_test.go

```diff
@@ -300,8 +301,8 @@ func TestAddSearchDelete(t *testing.T) {
 
 	// search rule1,rule2
 	repo.Mutex.RLock()
-	require.EqualValues(t, api.Rules{&rule1, &rule2}, repo.SearchRLocked(lbls1))
-	require.EqualValues(t, api.Rules{&rule3}, repo.SearchRLocked(lbls2))
+	assert.ElementsMatch(t, api.Rules{&rule1, &rule2}, repo.SearchRLocked(lbls1))
```

Comment:
If something goes wrong, the test will continue and will try to remove by label even if rules are not present. Do you want to keep assert?

Reply:
oh, good point, I was debugging something but forgot to switch them back